toolkit/tools/scheduler/schedulerutils/preparerequest.go

```diff
@@ -80,13 +92,6 @@ func canUseCacheForNode(pkgGraph *pkggraph.PkgGraph, node *pkggraph.PkgNode, pac
 		return
 	}
 
-	// If delta build enabled, then we can use the cache, unless a dependency of this package is rebuilding
-	if deltaBuild {
-		logger.Log.Warnf("Delta build: Using cached version of %v regardles of rebuilding dependencies", node.FriendlyName())
-		canUseCache = true
-		return
-	}
```

Comment:
I like removing code. I already lost track of how this used to work, so as long as Andrew is fine with this change and we got it covered in our build pipelines, then I'm happy.

Reply:
I'll double check with @anphel31 again, but this was the inconsistency we talked about with the delta feature as initially implemented. tldr this code path never actually did anything because we would never queue an implicit node for building (ie it would never check `canUseCacheForNode()`) until we had switched to the global implicit enable state anyways.